src/client/ui/PaylineGraphic.ts

```diff
@@ -0,0 +1,101 @@
+import * as PIXI from "pixi.js";
+
+export class PaylineGraphic extends PIXI.Container {
```

Comment:
You may want to make such components more abstract to be able to reuse them elsewhere.
For now, it is tightly tied to slot machine size, in namings and logic.
For example, that component could only accept a coordinate map of all possible points and a "draw" method with a list of points to draw lines between them. And then you position it accordingly wherever you created it.